src/jllvm/object/ClassObject.hpp

```diff
@@ -623,6 +623,11 @@ class ClassObject final : private llvm::TrailingObjects<ClassObject, VTableSlot>
         return getField(fieldName, fieldType, &Field::isStatic);
     }
 
+    Field* getStaticField(llvm::StringRef fieldName, FieldType fieldType)
+    {
+        return getField(fieldName, fieldType, &Field::isStatic);
```

Comment:
Should this not be a `std::mem_fn`?

Reply:
The implementation of `getField` uses `std::invoke` which is also able to use pointer to member functions directly, rather than requiring an adaptor such as `std::mem_fn`.